src/main/java/uk/gov/hmcts/reform/ucmc/config/PaymentsConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.ucmc.config;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Data
+@Configuration
```

Comment:
How about using `@ConfigurationProperties` ? i.e.

```suggestion
@Component
@ConfigurationProperties(prefix = "payments.api")
public class PaymentsConfiguration {
    private String siteId;
    private String service;
}
```

Reply:
It's a good idea, but it's not going to work with snake_case or kebab-case (and that's the convention for config properties), so I'd rather keep explicitly defined constructor.